3001-4000/LIT3592Miracle.xml

```diff
@@ -25,17 +29,30 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </sourceDesc>
       </fileDesc>
       <encodingDesc>
-         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
+            href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
             <xi:fallback>
                <p>Definitions of prefixes used.</p>
             </xi:fallback>
          </xi:include>
       </encodingDesc>
       <profileDesc>
          <abstract>
-            <p/>
+            <p> A miracle conflation appears in various manuscripts, linked to this generic miracle
```

Comment:
I suggest a modification of this abstract. You should 1) give a brief summary of the content and 2) inform the reader about the two existing versions and refer to them, 3) make clear, that this CAe is a general record and meant only for those attestations for which the exact version is not clear.